diff --git a/chromium/build/config/compiler/BUILD.gn b/chromium/build/config/compiler/BUILD.gn
index bad5e9f1822..f3a39121488 100644
--- a/chromium/build/config/compiler/BUILD.gn
+++ b/chromium/build/config/compiler/BUILD.gn
@@ -1898,7 +1898,7 @@ config("thin_archive") {
   # The macOS and iOS default linker ld64 does not support reading thin
   # archives.
   if ((is_posix && !is_nacl && (!is_apple || use_lld)) || is_fuchsia) {
-    arflags = [ "-T" ]
+    arflags = []
   } else if (is_win && use_lld) {
     arflags = [ "/llvmlibthin" ]
   }
