diff --git a/chromium/build/config/linux/pkg-config.py b/chromium/build/config/linux/pkg-config.py
index 2e38c7ffbd1..584b997653e 100755
--- a/chromium/build/config/linux/pkg-config.py
+++ b/chromium/build/config/linux/pkg-config.py
@@ -212,7 +212,7 @@ def main():
   includes = []
   cflags = []
   libs = []
-  lib_dirs = []
+  lib_dirs = [ "/app/lib" ]
 
   for flag in all_flags[:]:
     if len(flag) == 0 or MatchesAnyRegexp(flag, strip_out):
