diff --git a/src/core/web_engine_library_info.cpp b/src/core/web_engine_library_info.cpp
index 151918bd7..2e619e133 100644
--- a/src/core/web_engine_library_info.cpp
+++ b/src/core/web_engine_library_info.cpp
@@ -238,8 +238,7 @@ QString localesPath()
             candidatePaths << getResourcesPath(frameworkBundle()) % QDir::separator()
                             % QLatin1String("qtwebengine_locales");
 #endif
-            candidatePaths << QLibraryInfo::path(QLibraryInfo::TranslationsPath) % QDir::separator()
-                            % QLatin1String("qtwebengine_locales");
+            candidatePaths << "/app/translations/qtwebengine_locales";
             candidatePaths << fallbackDir();
         }
 
     if (!initialized) {
