diff --git a/src/core/web_engine_library_info.cpp b/src/core/web_engine_library_info.cpp
index 3a649227..56d31e46 100644
--- a/src/core/web_engine_library_info.cpp
+++ b/src/core/web_engine_library_info.cpp
@@ -205,7 +205,7 @@ QString localesPath()
 #if defined(OS_MAC) && defined(QT_MAC_FRAMEWORK_BUILD)
             getResourcesPath(frameworkBundle()) % QLatin1String("/qtwebengine_locales");
 #else
-            QLibraryInfo::location(QLibraryInfo::TranslationsPath) % QDir::separator() % QLatin1String("qtwebengine_locales");
+            "/app/translations/qtwebengine_locales";
 #endif
 
     if (!initialized) {
