diff --git a/src/core/web_engine_library_info.cpp b/src/core/web_engine_library_info.cpp
index 09a4141b..a2245ccc 100644
--- a/src/core/web_engine_library_info.cpp
+++ b/src/core/web_engine_library_info.cpp
@@ -201,12 +201,7 @@ QString subProcessPath()
 QString localesPath()
 {
     static bool initialized = false;
-    static QString potentialLocalesPath =
-#if defined(OS_MAC) && defined(QT_MAC_FRAMEWORK_BUILD)
-            getResourcesPath(frameworkBundle()) % QLatin1String("/qtwebengine_locales");
-#else
-            QLibraryInfo::location(QLibraryInfo::TranslationsPath) % QDir::separator() % QLatin1String("qtwebengine_locales");
-#endif
+    static QString potentialLocalesPath = "/app/translations/qtwebengine_locales";
 
     if (!initialized) {
         initialized = true;
