diff -Naur a/src/core/CMakeLists.txt b/src/core/CMakeLists.txt
--- a/src/core/CMakeLists.txt	2026-07-12 11:56:46.142749827 +0200
+++ b/src/core/CMakeLists.txt	2026-07-12 14:26:38.736077886 +0200
@@ -595,6 +595,10 @@
                     list(APPEND gnArgArg libyuv_use_sve=false)
                 endif()
             endif()
+            if(cpu MATCHES aarch64 OR cpu MATCHES "arm.*")
+                # https://bugreports.qt.io/browse/QTBUG-129985
+                list(APPEND gnArgArg build_webnn_with_xnnpack=false)
+            endif()
             unset(cpu)
         endif()
 
